FinancialTransactionsApi.Tests/V1/Controllers/FinancialTransactionsApiControllerTests.cs

```diff
@@ -27,6 +28,7 @@ public class FinancialTransactionsApiControllerTests
         private readonly Mock<IAddBatchUseCase> _addBatchUseCase;
         private readonly Mock<IGetTransactionListUseCase> _getTransactionListUseCase;
         private readonly Fixture _fixture = new Fixture();
+        private const string _token = "eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0.eyJuYW1lIjoidGVzdGluZyIsIm5iZiI6MTYzODQ2NTY3NiwiZXhwIjoyNTM0MDIyOTAwMDAsImlhdCI6MTYzODQ2NTY3Nn0.eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0";
```

Comment:
Is it secure using token in source code?

Reply:
It is fake one

Reply:
👍